src/PrestaShopBundle/Command/CheckTranslationDuplicatesCommand.php

```diff
@@ -112,9 +112,9 @@ protected function check($message1, $message2)
     protected function removeParams($message)
     {
         // Remove PrestaShop arguments %<arg>%
-        $message = preg_replace(PrestaShopTranslatorTrait::$regexClassicParams, '~', $message);
+        $message = preg_replace(Translator::$regexClassicParams, '~', $message);
```

Comment:
It seems equivalent but it could change over time 🤔 is it enforced by phpstan ?

Reply:
Constants can't be called from the trait. So I search a class using it. If we change it (remove trait from Translator, remove constant from trait), PHPStan will inform it.

Reply:
One more reason why I hate traits :sweat_smile:

Reply:
Approved.